### ISSUE TITLE
Falcon Textures become scrambled over a short period of time

### ISSUE THREAD

**Issue by Todd1215** [HeliosVirtualCockpit/Helios#598]
**Describe the Helios Issue**
If using the ILS training mission in BMS we have discovered that if you approach the airport in the training missions the Flying flag gets set to false and causes the Falcon Textures to become mangled and unusable. More investigation is needed before we can determine the cause.

**To Reproduce**
Steps to reproduce the behaviour:
Start Helios with a profile that contains the Helios Falcon Textures
start BMS and fly the ILS training mission
notice after several minutes that the MFDs change to some unusable image  

**Expected behavior**
That none of the Falcon Textures change after committed to 3D

**Screenshots**
This screen shot exhibits the issue
![om2hqCB](https://user-images.githubusercontent.com/6605805/182463861-6fc0e7b5-5db7-4f78-9f82-e5fd04cb24f4.jpg)

**please complete the following information:**
 - Helios Version [eg 1.6.5306.4]
 - Falcon BMS 4.36.1
 - [ ] Profile Editor
 - [ ] Control Center
 - [ ] Keypress Receiver
 - [ ] Something else (*remember to tell us what it is*)

**Comment by Todd1215** [comment]
After some further investigation I found that the Flying flag isn't getting set to false while in 3D. So that is not the issue. Also further testing revealed that flying other TE's didn't exhibit the same results as flying the ILS Landing TE. I even tried the ILS Weather TE thinking that it was something to do with the ILS TE's but I didn't get the issue in ILS weather TE at all.

**Comment by JacquesLees** [comment]
I was showcasing Falcon BMS with the Helios setup and with version 4.37 the HUD floating inside DED and MFD issue is becoming unbearable. It happened so often that I cannot use the MFD at all. We tried TE1, TE2, LGB, everytime right after several minutes into the mission.
It’s completely unable to do the LGB TE because of this. We found out that if we do some actions like lowering the gear MFD reappeared, only to be covered by floating HUD 2 secs later. I will upload the Helios logs file and reopen this issue.

**Comment by linknetx** [comment]
Reopened this issue for further investigation.

[Log Archive.zip](https://github.com/HeliosVirtualCockpit/Helios/files/10682567/Log.Archive.zip

 This are the 2 Helios logs and 1 BMS log, where it crashed, all while we were doing the Laser Guided Bomb TE. Floating HUD while we are preparing the mission and unable to use the MFD on Helios. We lowered the gear and it could bring the MFD for 2 seconds, then it went back to the floating HUD. We did it like 7 to 8 times and finally dicided to leave it and right after we hold down Pickle to be ready releasing the bomb it crashed.

**Comment by linknetx** [comment]
The ControlCenterProblems.log shows that Helios is still attempting to load the HeliosFalcon.dll version 1.6.5306.4 which was provided previously in an attempt to solve this issue. I suspect that the file has been renamed in an attempt to hide it. However if it still has the .dll filename extension then Helios will still try to load it whatever the filename. Can you please navigate to the Helios\Plugins folder and completely delete this renamed file if it exists. This should hopefully eliminate the exceptions shown in the Control Center log.

I don't think that this will solve the issues you are experiencing but at least it will remove one of the possibilities.

**Comment by JacquesLees** [comment]
Thank you very much Linknet, that's exactly the case, while I was checking yesterday I remembered this file but only checked my Document's Helios folder, didn't find Plugins so I thought I have already deleted it. Now that you mentioned it again I checked again on the main install location and both of the dll are present, one from 1.6.5306.4, and one from the fresh installed 1.6.5800.    
And I only renamed the file but it was still in dll extention, I will delete it now and have a try.
